src/components/Checkbox/Checkbox.js

```diff
@@ -32,62 +32,127 @@ export default class Checkbox extends React.Component {
       }
     };
 
-    const info = this.props.info ?
-      <div className={`${STAT_CLASSES.TITLE_HELP}`}
-      onClick={this.props.clickInfo}
-      onMouseEnter={this.props.mouseOverInfo}
-      onMouseLeave={this.props.mouseLeaveInfo}>
-        <button className={`${BUTTON_CLASSES.BUTTON} -icon -sm -flat`} aria-label="Help">
-          <i className={`${ICON_CLASS} chi-icon icon-circle-info-outline -icon--primary`}></i>
-        </button>
-      </div> : '';
+    let layoutOptions;
+    switch (this.props.layout) {
+      case 'inline':
+        layoutOptions = '-inline';
+        break;
+      case 'vertical':
+        layoutOptions = 'chi-col -w--12 -mb--1';
+        break;
+      case '2':
+        layoutOptions = 'chi-col -w--6 -mb--1';
+        break;
+      case '3':
+        layoutOptions = 'chi-col -w--4 -mb--1';
+        break;
+      case '4':
+        layoutOptions = 'chi-col -w--3 -mb--1';
+        break;
+      case '6':
+        layoutOptions = 'chi-col -w--2 -mb--1';
+        break;
+      default:
+        layoutOptions = 'chi-col -w--12 -mb--1';
+        break;
+    }
+
+    const info = this.props.info
+      ? (
+        <div
+          className={`${STAT_CLASSES.TITLE_HELP}`}
+          onClick={this.props.clickInfo}
+          onMouseEnter={this.props.mouseOverInfo}
+          onMouseLeave={this.props.mouseLeaveInfo}>
+          <button className={`${BUTTON_CLASSES.BUTTON} -icon -sm -flat`} aria-label="Help">
+            <i className={`${ICON_CLASS} chi-icon icon-circle-info-outline -icon--primary`}></i>
+          </button>
+        </div>
+      ) : '';
+
 
     Array(11).fill()
-        .forEach((_, i) => {
-          if (this.props[`label${i}`]) {
-            checkboxesToRender.push(
-                <div className={`chi-form__item ${this.props.inline ? '-inline' : ''}`} key={`checkbox-${i}`}>
-                  <div className="chi-checkbox">
-                    <input type="checkbox" className="chi-checkbox__input" disabled={this.props[`disabled${i}`]}
-                     checked={this.props[`checked${i}`]} onChange={(e) => {}} />
-                    <label onClick={(e) => {
-                      toggleCheckbox(e.target, i);
-                    }} className="chi-checkbox__label" htmlFor="checkbox1">{this.props[`label${i}`]}</label>
-                  </div>
-                </div>
-            );
+      .forEach((_, i) => {
+        if (this.props[`label${i}`]) {
+          let inlineDiv;
+          if (layoutOptions !== '-inline') {
+            inlineDiv = <div className="chi-form__item"></div>;
+          } else {
+            inlineDiv = null;
           }
-        });
+
+          checkboxesToRender.push(
+            <div className={`${layoutOptions === '-inline' ? 'chi-form__item -inline' : layoutOptions}`} key={`checkbox-${i}`}>
```

Comment:
```suggestion
            <div className={`this.props.layout === 'inline' || this.props.inline ? 'chi-form__item -inline' : layoutOptions}`} key={`checkbox-${i}`}>
```
This way you don't break the existing prototypes